components/FormComponents.js

```diff
@@ -86,13 +95,14 @@ function Select({
   return (
     <Group>
       <Label htmlFor={name}>{labelText}</Label>
-      <Selection className="noselect">
+      <Selection className="noselect" disabled={disabled}>
         <SelectionHeader
           className="noselect"
-          onClick={toggling}
+          onClick={disabled ? () => setIsOpen(false) : toggling}
```

Comment:
can you just use setIsOpen(false) here? I mean getting rid of the Callback thing's

Reply:
no. you would fall into a loop without callback